features/step_definitions/web_steps.rb

```diff
@@ -0,0 +1,82 @@
+Given /the following questions exist/ do |questions_table|
+  questions_table.hashes.each do |question|
+    Question.create(
+      title: question['title'],
+      options: eval(question['options']),
+      next_question: eval(question['next_question']),
+      can_skip: question['can_skip']
+    )
+  end
+end
+
+Then /(.*) seed questions should exist/ do |n_seeds|
+  Question.count.should be n_seeds.to_i
+end
+
+Given /^(?:|I )am on "(.+)"$/ do |page_name|
+  if page_name == "landing page"
+    visit landing_path
+  end
+  if page_name == "home page"
+    visit root_path
+  end
+end
+
+Then /I should see a sign in button/ do
+  expect(find_button("login-button").value).to eq("Placeholder Login")
```

Comment:
Oh yeah; you copied this from Juan's testing branch, right?
https://github.com/eilenshahbaz/LEAD-Center/pull/18/files/1c33da113fb7da020a813508540f3bf9ce7a37a1#r275173303
tl;dr: should probably check existence of button only, not the value of "Placeholder Login"
```suggestion
  expect(page).to have_button('login-button')
```